python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -385,46 +430,55 @@ async def run_cwfs(self):
         self.intra_exposure = await loop.run_in_executor(
             executor, self.get_isr_exposure, self.intra_visit_id
         )
+        
+        # create a clone to be used for detection
         self.detection_exp = self.intra_exposure.clone()
 
         self.extra_exposure = await loop.run_in_executor(
             executor, self.get_isr_exposure, self.extra_visit_id
         )
 
-        # Prepare detection exposure
+        # Prepare detection exposure, which adds two donuts together
         self.detection_exp.image.array += self.extra_exposure.image.array
         self.detection_exp.image.array -= np.median(self.detection_exp.image.array)
 
+        # Run source detection
         await loop.run_in_executor(executor, self.select_cwfs_source)
 
         await loop.run_in_executor(executor, self.center_and_cut_images)
+        
         if self.binning != 1:
-            self.log.info("Running CWFS code USING BINNED IMAGES.")
+            self.log.info(f"Running CWFS code using images binned by {self.binning} in each dimension.")
 
         # Now we should be ready to run cwfs
 
+        # reset inputs just incase 
         self.algo.reset(self.I1[0], self.I2[0])
 
+        # for a slow telescope, should be running in paraxial mode
         await loop.run_in_executor(
-            executor, self.algo.runIt, self.inst, self.I1[0], self.I2[0], "onAxis"
+            executor, self.algo.runIt, self.inst, self.I1[0], self.I2[0], "paraxial"
         )
-        # self.algo.runIt(self.inst, self.I1[0], self.I2[0], 'onAxis')
 
+        self.log.warning(" TODO: Negative sign being used infront of calculated Coma-X Zernike! Artifact tha appeared while checking at NCSA. Needs investigation on other platforms!")
```

Comment:
Maybe add a comment instead of a warning?

Reply:
Not sure.. this is a pretty serious issue...